test/test_helper.exs

```diff
@@ -25,4 +27,32 @@ defmodule Protobuf.TestHelpers do
 
     Map.put(context, :tmp_dir, tmp_dir_name)
   end
+
+  def get_type_spec_as_string(module, bytecode, type)
```

Comment:
What kind of sorcery is this? 😮

Reply:
I 100% did not steal this from the Elixir codebase. I'd never do that